source/agora/test/ManyValidators.d

```diff
@@ -216,7 +216,7 @@ unittest
 
         // check enrollment
         nodes.each!(n =>
-            retryFor(n.getEnrollment(enroll.utxo_key) == enroll, 5.seconds));
+             retryFor(n.getEnrollment(enroll.utxo_key) == enroll, 5.seconds));
```

Comment:
Leftover ?

Reply:
Will fix

Reply:
Also updated my VSC diff settings to not ignore whitespace changes so will be less likely to be missed next time.